app/process-tasks.js

```diff
@@ -39,12 +43,26 @@ function executeWorkerForTaskType (worker, task) {
 
   return worker.execute(task)
     .then(function () {
-      return completeTaskWithStatus(task.id, statusEnum.COMPLETE).then(function () {
+      return completeTaskWithStatus(task.id, taskStatus.COMPLETE)
+      .then(function () {
+        if (task.type === taskType.CALCULATE_WORKLOAD_POINTS) {
+          countTaskStatuses(task).then(function (totals) {
+            if (totals.numPending === 0 && totals.numInProgress === 0 && totals.numFailed === 0) {
+              updateWorkloadReportStatus(task.id, workloadReportStatus.COMPLETE)
```

Comment:
This should be the workload report ID rather than the task ID. Unit test should be updated to verify the correct ID is passed to `updateWorkloadReportStatus`